pkg/services/ngalert/notifier/multiorg_alertmanager_test.go

```diff
@@ -179,28 +179,28 @@ func TestMultiOrgAlertmanager_SyncAlertmanagersForOrgsWithFailures(t *testing.T)
 	{
 		require.NoError(t, mam.LoadAndSyncAlertmanagersForOrgs(ctx))
 		require.Len(t, mam.alertmanagers, 3)
```

Comment:
We no longer have `ready` as this was originally designed to be an internal non-safe version of `Ready` - this now lives in in `grafana/alerting`.

That aside, there's no reason why this test should rely on `ready` except to make it a tiny bit faster.